lib/utils/keymaps.js

```diff
@@ -0,0 +1,41 @@
+import {remote} from 'electron';
+import CommandRegistry from '../command-registry';
+
+const {getKeymaps} = remote.require('./config');
+
+export default function isCommands(e) {
+  let keys = [];
+
+  if (e.metaKey && process.platform === 'darwin') {
+    keys.push('cmd');
+  } else if (e.metaKey) {
+    keys.push(e.key);
+  }
+
+  if (e.ctrlKey) {
+    keys.push('ctrl');
+  }
+
+  if (e.shiftKey) {
+    keys.push('shift');
+  }
+
+  if (e.altKey) {
+    keys.push('alt');
+  }
+
+  if (e.key === ' ') {
+    keys.push('space');
+  } else if (e.key !== 'Meta' && e.key !== 'Control' && e.key !== 'Shift' && e.key !== 'Alt') {
+    keys.push(e.key.replace('Arrow', ''));
+  }
+
+  keys = keys.join('+');
+  const cmd = getKeymaps().keys[keys.toLowerCase()];
+
+  if (CommandRegistry.isCommands(cmd)) {
```

Comment:
It could be better to call this module `getCommands` and return cmd.
This `CommandRegistry.isCommands(cmd)` test and exec should maybe be done in `lib/hterm.js` (where current `isCommands` is called). In this way, command execution will not be hidden anymore and exec could be move elsewhere in the future, to not be linked too hardly with `hterm`.